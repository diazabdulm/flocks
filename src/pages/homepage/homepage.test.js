import React from "react";
import { shallow } from "enzyme";

import HomePage from "./homepage.component";

let wrapper;

describe("<HomePage />", () => {
  beforeEach(() => {
    wrapper = shallow(<HomePage />);
  });

  it("renders HomePage component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
