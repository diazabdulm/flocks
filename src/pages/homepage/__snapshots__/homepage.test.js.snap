// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> renders HomePage component 1`] = `ShallowWrapper {}`;
